Kiosk/Bid Fulfillment/RACFunctions.swift

```diff
@@ -22,19 +16,22 @@ func centsToPresentableDollarsString(cents:AnyObject!) -> AnyObject! {
     return ""
 }
 
-func isZeroLengthString(string:AnyObject!) -> AnyObject! {
+func isZeroLengthString(string: AnyObject!) -> AnyObject! {
     return countElements(string as String) == 0
 }
 
-func is4CharLengthString(string:AnyObject!) -> AnyObject! {
-    return countElements(string as String) == 4
+func isStringLengthIn(range: Range<Int>)(string: AnyObject!) -> AnyObject! {
```

Comment:
Nice, curry all the things! Saw your slides for Istanbul and they looked familiar :wink:.